assets/scripts/segments/EmptySegment.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import MeasurementText from '../ui/MeasurementText'
+import { TILE_SIZE } from '../segments/view'
+import { t } from '../app/locale'
+
+class EmptySegment extends React.PureComponent {
+  static propTypes = {
+    remainingWidth: PropTypes.number,
+    occupiedWidth: PropTypes.number,
+    units: PropTypes.number,
+    position: PropTypes.string
+  }
+
+  repositionEmptySegment = () => {
+    const { remainingWidth, occupiedWidth, position } = this.props
+    const segmentInfo = {
+      className: 'segment empty',
+      segmentWidth: 0,
+      widthValue: 0
+    }
+
+    if (remainingWidth > 0) {
+      const width = (occupiedWidth) ? (remainingWidth / 2 * TILE_SIZE) : (remainingWidth * TILE_SIZE)
+      if (!occupiedWidth && position === 'right') {
+        return segmentInfo
+      } else {
+        segmentInfo.segmentWidth = (position === 'right') ? width - 1 : width
+        segmentInfo.widthValue = width / TILE_SIZE
```

Comment:
I'm curious if we can simplify our use of `TILE_SIZE` here because it's effectively being multiplied in line 25 only to be removed in line 30. Could `TILE_SIZE` be removed in lines 25 and 30, and only be used in line 29, where it's needed to create the "render" value?